1920.build-array-from-permutation.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/build-array-from-permutation/
+ */
+
+/**
+ * @param {number[]} nums
+ * @return {number[]}
+ */
+var buildArray = function(nums) {
+    let result = [];
+    
+    for (let i = 0; i < nums.length; i++) {
+        result.push(nums[nums[i]]);
+    }
+    
+    return result;
+};
```

Comment:
You can also solve this problem by using Array.prototype.map() like this.
`return nums.map(num => nums[num])`